src/app/models/user.ts

```diff
@@ -1,7 +1,7 @@
 export interface User {
     name: string,
     email: string,
-    password: string,
+    password: string
```

Comment:
don't you have an error here because of that missing comma?

Reply:
I think it does not matter when this happens inside the definition of an interface. Not sure if it was used for this component, but I have changed it in order to be consistent.